localtypings/pxtarget.d.ts

```diff
@@ -40,13 +40,27 @@ declare namespace pxt {
         bannedRepos?: string[];
         allowUnapproved?: boolean;
         preferredRepos?: string[]; // list of company/project(#tag) of packages to show by default in search
+        preferredRepoLib?: RepoData[];
+        approvedRepoLib?: RepoData[];
         // format:
         // "acme-corp/pxt-widget": "min:v0.1.2" - auto-upgrade to that version
         // "acme-corp/pxt-widget": "dv:foo,bar" - add "disablesVariant": ["foo", "bar"] to pxt.json
         upgrades?: pxt.Map<string>;
         // list of trusted custom editor extension urls
         // that can bypass consent and send/receive messages
         approvedEditorExtensionUrls?: string[];
+        extensionsToIgnore?: string[]; // List of extensions to ignore when allowing for deletion
```

Comment:
nit: my reading of this / the code using looks like this field is used to mark 'extensions that we don't want to mark as isExtension', but my first thought was that it was 'extensions to hide from search / tag results' -- maybe 'unmarkedExtensions' or something similar to that? Not sure

Reply:
yeah it's being used for hiding for toolbox, but I think maybe it'd be good to use to hide from installed extensions too? Wanted to see if people had feedback about it during testing